d = {'a':1, 'b': 2}
print d['c']

'''
Traceback (most recent call last):
  File "exceptions_KeyError.py", line 13, in <module>
    print d['c']
KeyError: 'c'
'''